src/backend/Csrs.Interfaces.Dynamics/ModelExtensions/MicrosoftDynamicsCRMactivitypointer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel;
+
+namespace Csrs.Interfaces.Dynamics.Models
+{
+    class MicrosoftDynamicsCRMactivitypointerMetadata
```

Comment:
You do not need to add a metadata class if you are not providing any serialization customization.

Reply:
Hmm was wondering about that updated